programming/arduino/custom_firmware.md

```diff
@@ -12,6 +12,10 @@ On this page we talk specifically about Arduinos.
 However this page is applicable to opening any device that shows up as a serial port.
 </div>
 
+<div class="info">
+  This documentation refers to a feature which is only available on the physical robot kits.
```

Comment:
Have you considered "This feature is not available on the robot simulator", for clarity? 

(This and all instances below)

Reply:
I hadn't; this was copy/pasted from the various other places we say this sort of thing. Happy to change, but suggest we do that separately for speed?